### PR TITLE
Controllers for GET

### PR DESCRIPTION
**Pull request by nnk03**
Pull request for #2

### CODE REVIEW

File: src/backend/controllers/controllers.js
Comment (SyedMa3) [inline src/backend/controllers/controllers.js:1]: Define just the functions that will be called by the routes team.

Also use, `express-async-handler`.
Also, different file for different model. Like announcements.controllers.js

Comment (SyedMa3) [reply src/backend/controllers/controllers.js:1]: Also, contact models team to get to know about the models they defined.

**Review by SyedMa3 — CHANGES_REQUESTED** [review]

File: src/backend/controllers/announcements.controllers.js
Comment (SyedMa3) [inline src/backend/controllers/announcements.controllers.js:1]: You don't need to use router. You just have to define the contollers (functions) and export just them.

**Review by SyedMa3 — CHANGES_REQUESTED** [review]

**Comment by nnk03** [comment]
@SyedMa3 
Now, I have removed the routers and only defined the functions for announcements

File: src/backend/controllers/controllers.js
Comment (SyedMa3) [inline src/backend/controllers/controllers.js:1]: Delete this file if not needed

**Review by SyedMa3 — CHANGES_REQUESTED** [review]

**Review by SyedMa3 — APPROVED** [review]

**Pull request merged**